packages/editor/src/plugins/rows/components/plugin-menu-item.tsx

```diff
@@ -0,0 +1,65 @@
+import IconFallback from '@editor/editor-ui/assets/plugin-icons/icon-fallback.svg'
+import { EditorTooltip } from '@editor/editor-ui/editor-tooltip'
+import { EditorPluginType } from '@editor/types/editor-plugin-type'
+import { forwardRef } from 'react'
+
+import type { PluginMenuItemType } from '../contexts/plugin-menu/types'
+import { cn } from '@/helper/cn'
+
+interface PluginMenuItemProps {
+  item: PluginMenuItemType
+  selected: boolean
+  tooltipPosition?: 'right' | 'left'
+  onInsertPlugin: (pluginType: EditorPluginType) => void
+  onFocus: () => void
+  onBlur: () => void
+  onMouseMove: () => void
+  onMouseLeave: () => void
+}
+
+export const PluginMenuItem = forwardRef<
+  HTMLButtonElement,
+  PluginMenuItemProps
+>(function PluginMenuItem(
+  {
+    item,
+    selected,
+    tooltipPosition,
+    onInsertPlugin,
+    onFocus,
+    onBlur,
+    onMouseMove,
+    onMouseLeave,
+  },
+  ref
+) {
+  const { pluginType, title, icon, description } = item
+
+  const tooltipClassName = tooltipPosition
+    ? tooltipPosition === 'right'
+      ? '-left-0'
+      : '-right-0'
+    : '-left-24'
+
+  return (
+    <li>
+      <button
+        data-qa={`plugin-suggestion-${pluginType}`}
+        ref={ref}
+        onClick={() => onInsertPlugin(pluginType)}
+        onFocus={onFocus}
+        onBlur={onBlur}
+        onMouseMove={onMouseMove}
```

Comment:
@shn-srl do you see a downside to using `onMouseEnter` here?
(upsides: less events and I think the bug with focus stealing when the modal scrolls because of arrow key usage would be solved)

Reply:
@elbotho For context:
- https://github.com/serlo/frontend/pull/4017#issuecomment-2289112270
- https://github.com/serlo/frontend/pull/4017#issuecomment-2289214314